apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -28,40 +38,61 @@ const DepartureList = ({
   );
 
   let departures: DepartureInfo[] = [];
+  const groupedAlerts = alertsByRoute(alerts);
+  const alertsForRoute = groupedAlerts[route.id] || [];
 
+  const alertsByDirectionObj = alertsByDirectionId(alertsForRoute);
+  const alertsAffectingBothDirectionsArray = alertsAffectingBothDirections(
+    alertsForRoute
+  );
```

Comment:
thought (non-blocking): It seems like it might be tricky to remember to do both of these calls wherever we need to get all alerts by direction id. Maybe adding a helper like `allAlertsForDirection(directionId)` would simplify rather than needing to call both `alertsByDirectionId` and `alertsAffectingBothDirections`